static/js/components/exam/ConfirmToPearsonSiteDialog.js

```diff
@@ -0,0 +1,33 @@
+// @flow
+import React from 'react';
+import Dialog from 'material-ui/Dialog';
+
+import { dialogActions } from '../inputs/util';
+
+type ToPearsonSiteProps = {
+  open: boolean,
+  show: (open: boolean) => void,
+  openPearsonSiteLink: () => void
+}
+
+const ConfirmToPearsonSiteDialog = ({ open, show, openPearsonSiteLink }: ToPearsonSiteProps) => (
+  <Dialog
+    contentClassName="dialog"
+    className="dialog-to-pearson-site"
+    open={open}
+    onRequestClose={() => show(false)}
+    actions={dialogActions(() => show(false), () => openPearsonSiteLink(), false, "CONTINUE")}
+    autoScrollBodyContent={true}>
+    <div className="dialog-content">
+      <img src="/static/images/pearson_vue.png" width="200" height="70"/>
+      <h3 className="dialog-title">Test Registration is completed on the Pearson VUE website</h3>
+      <p>
+        I acknowledge that by clicking Continue I will be leaving the MicroMasters website and going to
+        the Pearson VUE website, and that I accept the Pearson VUE Group’s <a target="_blank"
+          href="https://home.pearsonvue.com/Legal/Privacy-and-cookies-policy.aspx">Terms of Service</a>.
+      </p>
+    </div>
+  </Dialog>
```

Comment:
this `Dialog` could be rolled into the existing `FinalExamCard` file, perhaps in the `cardWrapper` function or perhaps in a separate helper function.